lib/new_relic/agent.rb

```diff
@@ -436,6 +438,42 @@ def record_llm_feedback_event(trace_id:,
 
     # @!endgroup
 
+    # @!group LLM callbacks
+
+    # Set a callback proc for calculating `token_count` attributes for
+    # LlmEmbedding and LlmChatCompletionMessage events
+    #
+    # @param callback_proc [Proc] the callback proc
+    #
+    # Typically this method should be called only once to set a callback for
```

Comment:
Do we need the word 'typically'?

Reply:
I don't think so. This documentation was drawn from another one of our APIs. I'll remove!